examples/moodbot/data/stories.yml

```diff
@@ -0,0 +1,23 @@
+stories:
+
+- story: happy path
+  steps:
+  - intent: mood_great 
+  - action: utter_happy
+
+- story: sad path 1
+  steps:
+  - intent: mood_unhappy
```

Comment:
i actually wonder if this is the correct split here 🤔  because "mood_unhappy" is the direct response to the bots greeting (it says "hi how are you" or something)

Reply:
@akelad ,

I restored the greet->utter_greet as starting point of all the stories, and removed the corresponding rule.